apps/fulfillment/.env.template

```diff
@@ -1 +1,11 @@
-ORYX_FULFILLMENT_CLIENT_ID=fulfillment-client-local
+# The url where the backend is hosted.
+ORYX_FULFILLMENT_BACKEND_URL=
```

Comment:
It would be nice if we could align this env var with the one we use in storefront. reason behind is that this would allow us to reuse launchpad logic to create env vars. If you agree, let's create a ticket for this.

Reply:
Let's discuss it on catchup.

Reply:
It's impossible as long as we gonna have the same variable for storefront and fulfillment on github.